recipe/meta.yaml

```diff
@@ -1,65 +1,130 @@
     - cmake
     - ninja
     - make
   host:
-    - kokkos
-    - libblas
-    - liblapacke
-    - libgomp      # [linux]
-    - llvm-openmp  # [osx]
-    - pip
-    - pybind11
     - python
   run:
-    - ninja
-    - numpy
-    - pennylane >=0.19
-    - python
-
+    - pennylane-core
+    - pennylane-lightning-core
+    
 test:
   imports:
-    - pennylane_lightning
+    - pennylane
   commands:
     - pip check
+    - pl-device-test --help
+    - pl-device-test --device default.qubit --shots None --skip-ops
+    - pl-device-test --device default.qubit --shots 10000 --skip-ops
+    - pl-device-test --device lightning.qubit --shots None --skip-ops
+    - pl-device-test --device lightning.qubit --shots 10000 --skip-ops
+
   requires:
+    - setuptools <=64.0.2 # version requirement from cvxpy
     - pip
 
+outputs:
+
+  - name: pennylane-core
+    version: {{ version }}
+    build:
+      run_exports:
+        - {{ pin_subpackage('pennylane-core', max_pin='x.x') }}
+      entry_points:
+        - pl-device-test=pennylane.devices.tests:cli
+      script: cd "${SRC_DIR}/pennylane" && {{ PYTHON }} -m pip install . -vv
+      number: 0
+    requirements:
+      host:
+        - pip
+        - python
+      run:
+        - appdirs
+        - autograd
+        - autoray ==0.3.1
+        - cachetools
+        - cvxopt
+        - cvxpy
+        - networkx
+        - numpy <1.24
+        - opt_einsum
+        - python
+        - requests
+        - retworkx
+        - scipy
+        - semantic_version >=2.10
+        - toml
+
+  - name: pennylane-lightning-core
+    version: {{ version }}
+    build:
+      run_exports:
+        - {{ pin_subpackage('pennylane-lightning-core', max_pin='x.x') }}
+      number: 0
+      # skip: true  # [win and vc<14]
+      skip: true  # [win]
+      script: cd "${SRC_DIR}/pennylane-lightning" && {{ PYTHON }} -m pip install . -vv
+    requirements:
+      build:
+        - {{ compiler('c') }}
+        - {{ compiler('cxx') }}
+        # Needed for Windows
+        - cmake
+        - ninja
+        - make
+      host:
+        - libblas
+        - liblapacke
+        - libgomp      # [linux]
+        - llvm-openmp  # [osx]
+        - pip
+        - pybind11
+        - python
+        - setuptools <=64.0.2 # version requirement from cvxpy
+      run:
+        - ninja
+        - numpy
+        - python
+
 about:
-  home: https://github.com/PennyLaneAI/pennylane-lightning
-  summary: PennyLane-Lightning plugin
-  license: Apache-2.0
-  license_file: LICENSE
-  doc_url: https://pennylane-lightning.readthedocs.io/en/latest/
+  home: https://github.com/PennyLaneAI/pennylane
+  summary: PennyLane is a Python quantum machine learning library by Xanadu Inc.
+  license: Apache-2.0 AND MIT
+  # MIT is pennylane/grouping/graph_colouring.py
+  license_file: pennylane/LICENSE
+  doc_url: https://pennylane.readthedocs.io/en/stable/
 
 extra:
   recipe-maintainers:
     - vincentmr
     - BastianZim
+    - vincentmr
```

Comment:
Hey @vincentmr, your name is being added a second time here.

Reply:
Removed.